docs/questionSharing.md

```diff
@@ -35,6 +35,14 @@ To refer to a question from another course, use the question id (qid) prefixed b
 ]
 ```
 
+## Client and Server Files
+
+If a sharing course attempts to share a question which accesses client or server files associated with a course instance or an assessment, the question will not work as expected because the consuming course can not use it within the context of the sharing course's course intance or assessment.
+
+Questions that make use of `clientFilesQuestion`, `serverFilesQuestion`, and `serverFilesCourse` will work as expected. Using `clientFilesCourse` in a question is not supported at this time.
```

Comment:
Nit, I'd be inclined to swap the order of this and the above paragraph. I think it's generally better to tell people "these specific things work" first, then explain the exceptions after that?

Reply:
Seems very reasonable to me. 

Done.